compiler/optimizer/src/main/java/edu/snu/onyx/compiler/optimizer/pass/compiletime/annotating/DefaultParallelismPass.java

```diff
@@ -25,13 +25,14 @@
 import edu.snu.onyx.common.ir.vertex.executionproperty.ParallelismProperty;
 
 import java.util.List;
-import java.util.OptionalInt;
-import java.util.stream.Collectors;
 
 /**
  * Optimization pass for tagging parallelism execution property.
  */
 public final class DefaultParallelismPass extends AnnotatingPass {
```

Comment:
Doesn't this pass applied after every reshaping pass?
If so, decreasing parallelism of vertex after shuffle edge have to be done by separate pass.

Reply:
I've noticed that `PrimitiveCompositePass` does not run after every reshaping pass anymore.